src/server/api/routers/reviews.ts

```diff
@@ -0,0 +1,43 @@
+import { TRPCError } from "@trpc/server";
+
+import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+import { reviews } from "~/server/db/schema";
+import { db } from "~/server/db";
+import { reviewSaveSchema, reviewSubmitSchema } from "~/schemas/review";
+
+export const reviewsRouter = createTRPCRouter({
+  save: protectedProcedure
+    .input(reviewSaveSchema)
+    .mutation(async ({ input, ctx }) => {
+      try {
+        const userId = ctx.session.user.id;
```

Comment:
Could you also check if the current user is an organizer? I forgot about this. Happy bday :))

Reply:
Added and thank you :P